bin/src/_main.rs

```diff
@@ -1,621 +1,621 @@
-                                            handle_send_status(s)
-                                        }
-                                    },
-                                    Err(e) => handle_client_error(e, shutdown_tx).await,
-                                }
-                            }
-                            Err(e) => error!("Couldn't batch lines {:?}", e),
-                        }
-                    }
-                })
-                .await
-                .expect("Join Error")
-            }
-        });
-
-    // Concurrently run the line streams and listen for the `shutdown` signal
-    tokio::select! {
-        _ = lines_driver => {}
-        _ = retry_driver => {}
-        _ = &mut shutdown_rx => {
-            info!("Received shutdown request")
-        }
-        signal_name = get_signal() => {
-            info!("Received {} signal, shutting down", signal_name)
-        }
-    }
-}
-
-async fn check_startup_lease_status(
-    start_option: Option<&K8sLeaseConf>,
-    claimed_lease_ref: &mut Option<String>,
-    client: Kube_Client,
-) {
-    let max_attempts = match start_option {
-        Some(K8sLeaseConf::Attempt) => {
-            info!("Getting agent-startup-lease (making limited attempts)");
-            K8S_STARTUP_LEASE_RETRY_ATTEMPTS
-        }
-        Some(K8sLeaseConf::Always) => {
-            info!("Getting agent-startup-lease (trying forever)");
-            -1
-        }
-        _ => {
-            info!(
-                "Kubernetes cluster initialized, K8s startup lease set to: {:?}",
-                start_option
-            );
-            return;
-        }
-    };
-
-    let k8s_lease_api =
-        k8s::lease::get_k8s_lease_api(&std::env::var("NAMESPACE").unwrap(), client).await;
-    let mut attempts = 0;
-    while (max_attempts == -1) || (attempts < max_attempts) {
-        info!("Attempting connection: {}", attempts);
-        match get_available_lease(K8S_STARTUP_LEASE_LABEL, &k8s_lease_api).await {
-            Some(available_lease) => {
-                info!("Lease available: {:?}", available_lease);
-                k8s::lease::claim_lease(
-                    available_lease,
-                    std::env::var("POD_NAME").unwrap(),
-                    &k8s_lease_api,
-                    claimed_lease_ref,
-                )
-                .await;
-                break;
-            }
-            None => {
-                attempts += 1;
-                info!("No lease available at this time. Waiting 1 second...");
-                tokio::time::sleep(Duration::from_millis(1000)).await;
-            }
-        };
-    }
-}
-
-#[cfg(unix)]
-async fn get_signal() -> &'static str {
-    let mut interrupt_signal = unix::signal(unix::SignalKind::interrupt()).unwrap();
-    let mut quit_signal = unix::signal(unix::SignalKind::quit()).unwrap();
-    let mut term_signal = unix::signal(unix::SignalKind::terminate()).unwrap();
-
-    return tokio::select! {

```

Comment:
Seems like a small change. Wondering why the large git diff?

Reply:
Looks like whitespace, the previous version of the file must have had windows line endings, when I changed it and re cargo-fmt'd it they much have been stripped